README.md

```diff
@@ -1 +1,35 @@
-# ethics-in-data-viz
\ No newline at end of file
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
```

Comment:
Maybe remove reference to hour-long, since there are a variety of ways to run the workshop.